### PR TITLE
feat: custom import proto paths

### PR DESCRIPTION
**Pull request by moecasts**
新增  `-m` 选项，允许自定义 `import proto` 的地址。

### CODE REVIEW

File: src/libs/path.ts [hunk 1]
```diff
@@ -0,0 +1,42 @@
+import { existsSync } from 'fs';
+import path from 'path';
+import protobuf from 'protobufjs';
+
+/**
+ * oarse the import paths from the import string
```
Comment (JasonkayZK) [inline src/libs/path.ts]: Wrong comment, 'parse' maybe?

File: src/libs/path.ts [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+ * override the resolvePath function of protobufjs.Root
+ * include the import paths to resolve the proto file
+ */
+export function getResolveProtoPathsFunction(imports: string[] = []) {
+  return (origin: string, target: string) => {
+    var normOrigin = protobuf.util.path.normalize(origin),
+      normTarget = protobuf.util.path.normalize(target);
+
+    var resolved = protobuf.util.path.resolve(normOrigin, normTarget, true);
+    var idx = resolved.lastIndexOf("google/protobuf/");
```
Comment (JasonkayZK) [inline src/libs/path.ts:23]: Hard code is not recommended, maybe consider to use a constant instead?

File: src/libs/path.ts (hunk 1 above)
Comment (moecasts) [reply src/libs/path.ts]: yes

File: src/libs/path.ts (hunk 2 above)
Comment (moecasts) [reply src/libs/path.ts:23]: This is the source code copied from `protobuf.js`, and there will be no possibility of reuse here, so it should be fine to use literals?

FYI： https://github.com/protobufjs/protobuf.js/blob/547afa26f76e22e5463a17aec082b0b60cd951d8/cli/pbjs.js#L189

File: src/libs/path.ts (hunk 1 above)
Comment (moecasts) [reply src/libs/path.ts]: modified~

**Comment by JasonkayZK** [comment]
Thanks for your contribution. ❤️  You are so cool!

**Comment by JasonkayZK** [comment]
A new version: v1.1.3 has been released, feel free to try your new feature! 👍

**Pull request merged**